apps/ocp-plugin/deploy/helm/flightctl-ocp-ui/values.yaml

```diff
@@ -1,12 +1,17 @@
-flightctlUi:
-  namespace: flightctl
-  image: quay.io/flightctl/flightctl-ocp-ui:latest
-  flightctlServer: https://flightctl-api:3443
-  flightctlMetricsServer: https://flightctl-api:9090
-  flightctlGrpcServer: flightctl-api-agent-grpc:7444
-  bootcImgUrl: "" #Bootc device image. Example: quay.io/example/example-agent-centos:bootstrap
-  qcow2ImgUrl: "" #Qcow2 device image. Example: https://example.com/disk.qcow2
-  certs:
-    ca: |
-      -----BEGIN
-      -----END
+## @param image.image Image to deploy.
```

Comment:
let's document also any global consumed values.

Reply:
This one is still pending in the same way we have for the other values:

https://github.com/flightctl/flightctl-ui/pull/112/files#diff-9712335e7da43c32d9689c43790084c6645fdaa287062d9f2b4eac266dbc07e1R6

Reply:
Or we don't need any of those at all?

Reply:
Not at the moment. Maybe later we will add some, but currently I dont see any that would be useful here.